### PR TITLE
Touch enabled

### PR DESCRIPTION
**Pull request by irongollem**
This patch adds touch listeners for the drag and drop handle. Resize doesn't yet work. Will try and implement Pinch for that next

### CODE REVIEW

**Comment by dattn** [comment]
Thank you for this pull request.

Could you please remove the dist folder from git, and add an "prepublishOnly" script to the package.json to force a build before publishing the package.
Instead of removing the dist folder from the .gitignore file, add  an .npmignore file with the same content as the .gitignore file but with the dist folder removed.

Why did you remove the package-lock file from the repo ?

**Comment by irongollem** [comment]
Hi Dattn, this is my first pr over github so didn't account for the system updating the merge request on its own. I've made some changes (like the temporary dist and package-lock) for local usage while waiting for this one to update :), aparently those now ended up in the PR as well :) 

I'll adjust the file to properly reflext a PR.

File: src/demo/index.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import Vue from 'vue'
 import App from './App'
+import VueTouch from 'vue-touch'
+
+Vue.use(VueTouch, {name: 'v-touch'})
```
[the remark this answers is not in the document]
Comment (irongollem) [reply src/demo/index.js]: Nowhere (anymore) and Ill remove it

**Comment by irongollem** [comment]
@dattn Not sure how to undo the package-lock deletion, but other than that I think this should be alright?

**Comment by irongollem** [comment]
@dattn can you review my last changes and if sufficient merge?

**Comment by dattn** [comment]
Sorry for the long wait, i will take a look tonight.

**Pull request merged**